examples/render-helm-chart-crds/README.md

```diff
@@ -0,0 +1,64 @@
+# render-helm-chart: Remote Chart with CRDs
+
+### Overview
+
+This example demonstrates how to imperatively invoke the `render-helm-chart`
+function to render a helm chart that contains CRDs in the templated output.
+
+### Function invocation
+
+First, let's render a terraform chart without CRDs:
+
+```shell
+$ kpt fn eval --image gcr.io/kpt-fn/render-helm-chart:unstable --network -- \
+name=terraform \
+repo=https://helm.releases.hashicorp.com \
+version=1.0.0 \
+releaseName=terraforming-mars 
+```
+
+You should see rendered chart resources in your local filesystem. You can run the
+following command to see what you have:
+
+```shell
+$ kpt pkg tree
+├── [configmap_terraforming-mars-terraform-test.yaml]  ConfigMap terraforming-mars-terraform-test
+├── [pod_terraforming-mars-terraform-test.yaml]  Pod terraforming-mars-terraform-test
+├── [role_terraforming-mars-terraform-sync-workspace.yaml]  Role terraforming-mars-terraform-sync-workspace
+├── [rolebinding_terraforming-mars-terraform-sync-workspace.yaml]  RoleBinding terraforming-mars-terraform-sync-workspace
+├── [serviceaccount_terraforming-mars-terraform-sync-workspace.yaml]  ServiceAccount terraforming-mars-terraform-sync-workspace
+├── [workspace_terraforming-mars-terraform-test.yaml]  Workspace terraforming-mars-terraform-test
+└── default
```

Comment:
I'm curious why there is a directory here? Is this because the original template has it?

Reply:
I am actually not sure why this directory is here. I am using the same example as the kustomize test (https://github.com/kubernetes-sigs/kustomize/blob/master/plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator_test.go). I think it is using this chart https://github.com/hashicorp/terraform-helm. I am reading through the chart to see if I can understand why it creates this directory.